ext/web/06_streams.js

```diff
@@ -658,7 +658,9 @@
    * @returns {ReadableStream<Uint8Array>}
    */
   function readableStreamForRid(rid, unrefCallback) {
-    const stream = new ReadableStream({
```

Comment:
something to note: ReadableStream initialization triggers a pretty serious deopt in V8. From a flamegraph of the send file server on `main`:

![image](https://user-images.githubusercontent.com/34997667/185556060-c0e8155d-8397-47bd-aae0-4c54dc987e27.png)